wurst/objediting/presets/ChannelAbilityPreset.wurst

```diff
@@ -5,6 +5,8 @@ import public ObjectIds
 import public ObjectIdGenerator
 import OrderStringFactory
 import BitSet
+import HashMap
```

Comment:
why?

Reply:
I was about to map strings to ability ids, then I realized it's a bad idea xD